### PR TITLE
Update stacks documentation

### LINKED ISSUE #50: Hard to understand the contains section on Stacks page
The "Contains" section on the stacks page could be better organized to indicate what's in the build/run images for a given stack. Perhaps we move this into a table to make it easier to read.

image

### PR DESCRIPTION
**Pull request by fg-j**
Thanks for contributing. To speed up the process of reviewing your pull request
please provide us with:

* A short explanation of the proposed change:
Resolves #50

The stacks documentation was unclear and out of date. I've updated this to include the latest information. Soon-to-come stacks are hidden in a commented-out state in the file.

Please confirm the following:
* [x] I have viewed, signed, and submitted the Contributor License Agreement.

* [ ] (For UX change) I have made sure this feature works well on both mobile- and desktop-sized screens.
* [ ] (For UX change) I have followed BEM naming conventions for my CSS (see README).

* [x] (For docs change) I have proofread for spelling, grammar, and accuracy.
* [ ] (For docs change) I have used `{{< ref >}}` and `{{< relref >}}` shortcodes for internal links (see Hugo [docs](https://gohugo.io/content-management/cross-references/)).
* [x] (For docs change) I have checked that all the links I added point to the intended content.
* [ ] (For docs change) I have tested that all code samples run correctly as written.

### CODE REVIEW

File: content/docs/concepts/stacks.md [hunk 1]
```diff
@@ -13,86 +13,79 @@ In the Getting Started tutorial, you used the Paketo builder to build a Node.js
...
-
-#### Contains:
-* Build: ubuntu:bionic + many common C libraries and utilities
-* Run: ubuntu:bionic + many common libraries and utilities
+The Paketo project releases several stacks. We currently support two Ubuntu
+distributions: 18.04 Bionic Beaver and 22.04 Jammy Jellyfish. Tiny, Base, and
+Full stack variants differ in the number of packages installed in the OS layer.
+The available stacks, _**from smallest to largest**_, are:
+- [Bionic Tiny](https://github.com/paketo-buildpacks/bionic-tiny-stack):
+  - Build image based on Ubuntu 18.04 Bionic Beaver; run image comparable to [distroless](https://github.com/GoogleContainerTools/distroless)
```
Comment (TisVictress) [inline content/docs/concepts/stacks.md:27]: @fg-j Looks like `distroless` is causing the spellcheck to fail. Isn't there a tag we can use hear to skip spellcheck?

Comment (fg-j) [reply content/docs/concepts/stacks.md:27]: I've added this word to the dictionary, since it's a "real noun" in this context.

**Comment by ForestEckhardt** [comment]
@fg-j I am a little confused by this PR would you want this merged now or are we still waiting for some functionality to be implemented?

**Comment by fg-j** [comment]
It can be merged as-is now. As/when more stacks and builders are available, it can be updated further.

**Review by ForestEckhardt — APPROVED** [review]

**Pull request merged**